src/engraving/rw/read410/tread.cpp

```diff
@@ -796,6 +799,18 @@ void TRead::read(Expression* expr, XmlReader& xml, ReadContext& ctx)
     }
 }
 
+void TRead::read(Footnote* expr, XmlReader& xml, ReadContext& ctx)
+{
+    while (xml.readNextStartElement()) {
+        const AsciiStringView tag = xml.name();
+        if (tag == "snapToDynamics") {
+            readProperty(expr, xml, ctx, Pid::SNAP_TO_DYNAMICS);
```

Comment:
I don't think this property is applicable for footnotes, is it? (same in read460)

Also, you only need to update read460; scores from older versions won't contain footnotes anyway, so no need to handle them here.